README.rst

```diff
@@ -193,7 +192,7 @@ If you want to run local integration tests, then use:
 
 ::
 
-    # Required arguments for integration tests are found in test/integ/conftest.py
+    # Required arguments for integration tests are found in test/conftest.py
 
     pytest test/integ --docker-base-name <your_docker_image> \
```

Comment:
should this be "pytest test/integ/local"

Reply:
same on line 206. github does not let me comment there -__-

Reply:
good catch; fixed